lib/install/rust.js

```diff
@@ -477,13 +477,13 @@ module.exports.cli = {
 // Loging function that checks if all rust components are installed.
 module.exports.checkSdk = () => {
   return exports.getBuildConfig()
-    .catch(e => {
+    .catch(e => new Promise((resolve, reject) => { // jshint ignore:line
       log.error('Could not find all the components for cross-compiling Rust:');
       log.error(e.message);
       log.info('Please run "cargo tessel sdk install" and try again.');
       log.info('To instead use the remote Rust compiler, use `t2 run <target> --rustcc`.');
-      process.exit(1);
-    });
+      reject();
```

Comment:
`reject(...)` is never called with no arguments. 

To reject with a verbose message, compose the contents and pass it along to `reject(...)`, something like this:  

```js
module.exports.checkSdk = () => {
  return new Promise((resolve, reject)) {
    exports.getBuildConfig().then(resolve).catch(error => {
      var message = tags.stripIndent`
        Could not find all the components for cross-compiling Rust:
        ${error.message}
        Please run "cargo tessel sdk install" and try again.
        To instead use the remote Rust compiler, use "t2 run <target> --rustcc".`;

      reject(message);
    });
  });
};
```